devnet/tx-verifier/Dockerfile.tx-verifier-evm

```
@@ -0,0 +1,23 @@
+# We need the built guardiand image
+FROM guardiand-image as guardiand
```

This change passes the pin check for me locally.
```suggestion
FROM guardiand-image AS base
```